db/db_impl_write.cc

```diff
@@ -170,15 +173,13 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
       }
     }
 
-    uint64_t last_sequence = versions_->LastSequence();
```

Comment:
Moving simply because I need `last_seqence` when I call `versions_->SetLastSequence` at the end.